docker-storage-setup.1

```diff
@@ -31,14 +31,11 @@ functionality to work properly.
 \f[B]Supported options for the configuration file\f[]:
 
 STORAGE_DRIVER:
-      Specify a storage driver to be used with docker. Default
-      driver is "devicemapper". Other valid values are overlay,
-      overlay2 and "". Empty string tells docker-storage-setup to
-      not do any storage setup.
+      Specify a storage driver to be used with container runtime. The default driver is "devicemapper". Valid values are overlay, overlay2 and "". Empty string tells docker-storage-setup to not perform any storage setup.
```

Comment:
I am trying to cleanup the format.  I had lots of problems.  Take another look at the format now.